app/webpack.common.ts

```diff
@@ -88,6 +88,10 @@ export const renderer = merge({}, commonConfig, {
         test: /\.(jpe?g|png|gif|ico)$/,
         use: ['file?name=[path][name].[ext]'],
       },
+      {
+        test: /\.cmd$/,
+        loader: 'file-loader',
```

Comment:
@niik if you've got any better ideas to satisfy webpack for this corner case instead introducing another loader I'd love to hear it, but basically this is where I ended up:

1. `fs-admin` has a Windows-specific `.cmd` script that it uses to `robocopy` directories
2. this is referenced in source like this: `require.resolve('./src/copy-folder.cmd')`
3. webpack complains when it encounters this code

```
[WEBPACK] Errors building renderer.js
Module parse failed: Unexpected token (1:3)
You may need an appropriate loader to handle this file type.
| if exist %2 rmdir %2 /s /q
| (robocopy %1 %2 /e) ^& IF %ERRORLEVEL% LEQ 1 exit 0
|
```

4. I thought I'd use the step above (with the other static files) but that didn't seem to make the webpack error go away
5. [`file-loader`](https://www.npmjs.com/package/file-loader) seemed to do the trick